UniqueList/UniqueList/UniqueList.cs

```diff
@@ -0,0 +1,37 @@
+namespace UniqueList;
+using Exceptions;
+public class UniqueList: UsualList
+{
+    public bool ValueAtList(int value)
+    {
+        if (Count == 0) {throw new EmptyListException("empty list");};
```

Comment:
```suggestion
        if (Count == 0) {
            throw new EmptyListException("empty list");
        }
```
условие if должно быть расположено на отдельной строке